console/console-init/ui/mock-console-server/README.md

```diff
@@ -416,89 +416,115 @@ query single_address_with_links_and_metrics {
 }
 ```
 
-## Retrieve all Iot projects
+## Retrieve list of Iot tenants and messaging projects
 
 ```
-query allProjects {
-  allProjects(
-    projectType: iotProject
-  ) {
+query {
+  allProjects {
     total
-    iotProjects {
-      metadata {
-        name
-        namespace
-        creationTimestamp
+    objects {
```

Comment:
We would also need a property that signifies if a project is 'Ready'|'Configuring'|'Pending'|'Failed'

Reply:
I would expect that you'll get access to the .status section of both underlying objects.